server/supernova/audio_backend/portaudio_backend.hpp

```diff
@@ -69,6 +69,11 @@ class portaudio_backend : public detail::audio_backend_base<sample_type, float,
 
     uint32_t get_latency(void) const { return latency_; }
 
+private:
+    // these values will be clipped to the number of available hardware ins/outs
+    uint16_t mHwInputChannels = 0;
```

Comment:
rename to `m_hwInputChannels` per [webkit naming conventions](https://webkit.org/code-style-guidelines/#names-data-members)